crates/compiler/test_gen/src/gen_tags.rs

```diff
@@ -2094,17 +2094,17 @@ fn nullable_wrapped_with_non_nullable_singleton_tags() {
 }
 
 #[test]
-#[cfg(any(feature = "gen-llvm", feature = "gen-wasm"))]
+#[cfg(any(feature = "gen-llvm", feature = "gen-wasm", feature = "gen-dev"))]
```

Comment:
```suggestion
```

At this point, can we get rid of the whole `#cfg` here? Is there any situation where it doesn't run? 😄

Reply:
there is sometimes, e.g. for llvm wasm. we may need them again for x86 vs aarch dev backend. I'd prefer to keep them for this PR at least